cloud_function/migrate_database.py

```
@@ -0,0 +1,586 @@
+                "name": "[WARNING] Errors Encountered",
+                "value": f"```\n{error_text}\n```",
+                "inline": False
+            })
+        
+        payload = {"embeds": [embed]}
+        
+        response = requests.post(DISCORD_WEBHOOK_URL, json=payload, timeout=10)
+        response.raise_for_status()
+        log(f"[OK] Discord notification sent: {title}")
+        
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)